src/main/java/com/xiaomi/infra/pegasus/rpc/interceptor/InterceptorManger.java

```diff
@@ -22,14 +26,22 @@ private InterceptorManger register(TableInterceptor interceptor) {
 
   public void before(ClientRequestRound clientRequestRound, TableHandler tableHandler) {
     for (TableInterceptor interceptor : interceptors) {
-      interceptor.before(clientRequestRound, tableHandler);
+      try {
+        interceptor.before(clientRequestRound, tableHandler);
+      } catch (PException e) {
+        logger.warn("interceptor-before execute failed!", e);
```

Comment:
```suggestion
        logger.warn("interceptor-before execute failed!", e);
```

I think this is a serious error that should be thrown out to the users. Catching the exception internally is not appropriate.